src/main/java/rwg/commands/RwgNoiseCommand.java

```diff
@@ -0,0 +1,89 @@
+package rwg.commands;
+
+import java.util.Locale;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.WrongUsageException;
+import net.minecraft.util.ChatComponentText;
+import rwg.util.NoiseGeneratorWrapper;
+import rwg.util.NoiseImplementation;
+import rwg.world.RwgWorldSavedData;
+
+public class RwgNoiseCommand extends CommandBase {
+    @Override
+    public String getCommandName() {
+        return "rwg_noise";
+    }
+
+    @Override
+    public String getCommandUsage(ICommandSender sender) {
+        return "/rwg_noise get|opensimplex|perlin";
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) {
+
+        if (args.length < 1) {
+            throw new WrongUsageException(this.getCommandUsage(sender));
+        }
+
+        NoiseImplementation current = RwgWorldSavedData.getNoiseImplementation();
+
+        if (args[0].equals("get")) {
+            sender.addChatMessage(new ChatComponentText("Current noise type is " + current));
```

Comment:
Same here, replace ChatComponentText with the localizable variant and put the messages into the language file.